### DISCUSSION TITLE
Student Developer Pack Application not making Progress

### DISCUSSION THREAD

**Discussion by VinayR-GitHub** [community/community#114587] [GitHub Education] [answered]
### Hi everyone,

I recently applied for the GitHub student developer pack. However, I have not been approved, or rejected, as I cannot find a rejection message where one should be, and I get the notification that my request is "still being processed". It has been like this for months. 
What should I do in order to get my request approved?

**Comment by mairinkdev** [comment, marked as answer]
Contact GitHub Support, Reach out to GitHub Support and inquire about the status of your application. Provide them with any relevant information, such as your application details or any confirmation emails you may have received

Patience, Lastly, while waiting for a response from GitHub, try to be patient. Sometimes, these processes can take time due to the volume of applications they receive

**Reply by nirdidev05** [reply]
do you have the adresse email of github support

**Reply by mairinkdev** [reply]
https://docs.github.com/pt/support/contacting-github-support
